fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1709,98 +1709,93 @@ private Long getLongSqlRowSet(final SqlRowSet rs, final String column) {
         return val;
     }
 
-    private String dataScopedSQL(final String appTable, final Long appTableId) {
```

Comment:
why to remove the `final` from these variables?

Reply:
It was my mistake. Will correct it. Thanks